moto/elbv2/models.py

```diff
@@ -1308,41 +1319,23 @@ def modify_listener(
 
             # HTTPS checks
             if protocol == "HTTPS":
-                # HTTPS
-
-                # Might already be HTTPS so may not provide certs
-                if certificates is None and listener.protocol != "HTTPS":
-                    raise RESTError(
-                        "InvalidConfigurationRequest",
-                        "Certificates must be provided for HTTPS",
-                    )
-
                 # Check certificates exist
-                if certificates is not None:
-                    default_cert = None
-                    all_certs = set()  # for SNI
-                    for cert in certificates:
-                        if cert["is_default"] == "true":
-                            default_cert = cert["certificate_arn"]
-                        try:
-                            self.acm_backend.get_certificate(cert["certificate_arn"])
-                        except Exception:
-                            raise RESTError(
-                                "CertificateNotFound",
-                                "Certificate {0} not found".format(
-                                    cert["certificate_arn"]
-                                ),
-                            )
-
-                        all_certs.add(cert["certificate_arn"])
-
-                    if default_cert is None:
```

Comment:
All this is redundant cause of what is in the AWS docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listener.html#cfn-elasticloadbalancingv2-listener-certificates

> You must provide exactly one certificate if the listener protocol is HTTPS or TLS.